templates/generic_show_form.html.twig

```diff
@@ -519,7 +521,7 @@
                         ) }}
                      {% endif %}
 
-                     {% if item.isField('size') %}
+                     {% if item.isField('size') and item_type != 'DatabaseInstance' %}
```

Comment:
IMHO, having this kind of specific code here is a bad idea. I think `size` is not a generic field, so it should probably be removed from here and moved in dedicated `Item_DeviceMemory`, `Monitor` and `Database` forms. These are the only GLPI core items that have this field.

Reply:
On the other hand, what is this field used for in DatabaseInstance? The `prepareInputForAdd` forcibly removes the field from the input, the form doesn't have an input for it, there is no search option, and the inventory feature code doesn't seem to have anything related to it. Looks like it was maybe just a copy-paste from `Database` and the column could actually be dropped.

Reply:
Maybe it could be dropped, indeed.